SESSION 1 - VISUALIZATION - FEB 25

Malekikh opened the pull request:
Add the codes practiced during the session along with the last activity

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?

## A reference to a related issue in your repository (if applicable)

## Checklist
- [ ] I can confirm that my changes are working as intended

Malekikh committed:
SESSION 1 - VISUALIZATION - FEB 25

Add the codes practiced during the session along with the last activity

Malekikh committed:
COMMENTS ADDED DURING THIS SESSION

Malekikh committed:
LIVE CODE SESSION 3 - FEB 27

amanda-ng518 reviewed (changes requested):
week 2 codes missing

Malekikh committed:
Visualizations from live code
It seems like my live codes are submitted via multiple branches - resending them all via assignment_1 pull request

amanda-ng518 reviewed (approved):
A1 complete
